frontend/src/pages/Invite/AddCompany.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { Section } from '@digicatapult/ui-component-library'
+import { ContainerStyle, HeadingText, Button } from '../../components/shared'
+import ButtonWithImage from '../../components/ButtonWithImage'
+
+const InviteCompany = () => {
+  return (
+    <ContainerStyle>
+      <Section
+        margin={'40px 0px 0px 0px'}
+        headingLevel={1}
+        background={'#FFF'}
+        title={'- NICE Network - '}
+      ></Section>
```

Comment:
should be possible to replace these 2 components with the `Header` component like here https://github.com/digicatapult/nice-agent-portal/blob/e39fdf6b648086c607d4bd1298d1a3d433e3f952/frontend/src/pages/Issue/CompanyDetailsForm.js#L24

Reply:
oh yes I forgot! Thank you